package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vscode-test-explorer",
-  "version": "2.18.1",
+  "version": "2.19.1",
```

Comment:
I'm not sure why this change appeared -- maybe the bump to 2.19.1 in https://github.com/hbenl/vscode-test-explorer/commit/b073db89610f7083e7aa38dec3030522d1a76a52 missed this?